packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.jsx

```diff
@@ -591,7 +600,10 @@ const AccountTransferForm = ({
                                             !!selected_to.error ||
                                             !+selected_from.balance ||
                                             !!converter_from_error ||
-                                            !!converter_to_error
+                                            !!converter_to_error ||
+                                            (selected_from.currency === selected_to.currency
+                                                ? !values.amount
+                                                : !converter_from_amount)
```

Comment:
@george-usynin-binary Can we move this logic for `is_disabled` to a separate function? 🙏🏻